app/views/evaluator_submission_assignments/_unassign_evaluator_submission_modal.html.erb

```diff
@@ -1,17 +1,17 @@
 <dialog
   id="unassign-evaluator-submission-modal"
   data-unassign-evaluator-submission-modal-target="modal"
-  aria-labelledby="modal-1-heading"
-  aria-describedby="modal-1-description"
+  aria-labelledby="unassign-modal-heading"
```

Comment:
NIT: don't change anything here, but some of these view changes are unrelated to the current PR and it makes it harder to review everything together. in the future please try to keep the scope of PRs tight on the tickets at hand to make reviews easier and quicker.